tool/update_version.dart

```diff
@@ -140,6 +154,28 @@ void writeVersionToChangelog(File changelog, String version) {
   ].joinWithNewLine());
 }
 
+void writeVersionToIndexHtml(
+  File indexHtml,
+  String oldVersion,
+  String newVersion,
+) {
+  final lines = indexHtml.readAsLinesSync();
+  final revisedLines = <String>[];
+  for (final line in lines) {
+    if (line.contains(oldVersion)) {
```

Comment:
make this more robust. If the file doesn't contain the oldVersion we need to throw an exception as we failed to update.